cfpb/cfgov-refresh
# Upgrade cfpb-chart-builder to v6 with Highcharts 7 upgrade

Includes the following releases from cfpb-chart-builder:
- v6.0.0: Upgrade to Highcharts 7  https://github.com/cfpb/cfpb-chart-builder/releases/tag/v6.0.0
- v6.0.1: Enable styledMode for CSS use with GeoMap chart https://github.com/cfpb/cfpb-chart-builder/releases/tag/v6.0.1

## Additions

-

## Removals

-

## Changes

-

## Testing

1. `./frontend.sh`
1. `./runserver.sh`
1. Visit http://localhost:8000/data-research/mortgage-performance-trends/mortgages-90-or-more-days-delinquent/ and compare with [live site](https://www.consumerfinance.gov/data-research/mortgage-performance-trends/mortgages-90-or-more-days-delinquent/). All functionality and styling should be the same.
1. Visit http://localhost:8000/data-research/consumer-credit-trends/credit-cards/borrower-risk-profiles/ and compare with [live site](https://www.consumerfinance.gov/data-research/consumer-credit-trends/credit-cards/borrower-risk-profiles/). 
## Screenshots


## Notes

-

## Todos

-

## Checklist


## Testing checklist

### Browsers


### Accessibility


### Other
